sections/03_categorize.md

```diff
@@ -238,12 +261,90 @@ Ratner, Bach, and Ré described labeled training data as "The _New_ New Oil"
 framing, data are abundant and not a scarce resource. Instead, new approaches
 to solve problems arise when labeled training data become sufficient to enable
 them. Based on our review of research on deep learning methods to categorize
-disease, the latter framing rings true. We expect each of these approaches to
+disease, the latter framing rings true.
+
+In addition to methodological improvements, a robust culture of data sharing -
+and in particular the sharing of high-quality labeled datasets - would do much
+to speed advances in this domain. The cultural barriers are perhaps best
+captured by the implications of using the term "research parasite" to describe
+scientists who use data from other researchers [@doi:10.1056/NEJMe1516564]. In
+short, a field that honors only discoveries and not the hard work of generating
+useful data will have difficulty encouraging scientists to share their hard-won
+data. Unfortunately, it's precisely those data that would help to power deep
+learning in the domain. Though not a methodological consideration, are underway
+to recognize those who promote an ecosystem of rigorous sharing and analysis
+[@doi:10.1038/ng.3830].
```

Comment:
Yea the API call appears to be working:

```
curl --location \
  --header "Accept: application/vnd.citationstyles.csl+json" \
  https://doi.org/10.1038/ng.3830
```

Let me rebuild. If not resolved, I'll investigate.